index.js

```diff
@@ -216,8 +226,18 @@ module.exports = function(options) {
       fn.metricsQueue = []
       let args = [].slice.call(arguments)
 
+      /* Only resolve DNS on coldstarts */
```

Comment:
this comment got reworded in the rebase... it should be "resolve on every invocation" (that is the behavior here)